src/examples/typography/Lists.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { OrderedList, UnorderedList } from '@zendeskgarden/react-typography';
+
+const marginBottom = `margin-bottom: 18px;`;
+
+const Example = () => (
+  <>
+    <OrderedList size="small" css={marginBottom}>
+      <OrderedList.Item>
+        The world&apos;s tallest-growing tree is the coast redwood
+      </OrderedList.Item>
+      <OrderedList.Item>Bamboo can grow 35 inches in a single day</OrderedList.Item>
+      <OrderedList.Item>During the 1600s, tulips were worth more than gold</OrderedList.Item>
+    </OrderedList>
+    <OrderedList size="medium" css={marginBottom}>
+      <OrderedList.Item>
+        The world&apos;s tallest-growing tree is the coast redwood
+      </OrderedList.Item>
+      <OrderedList.Item>Bamboo can grow 35 inches in a single day</OrderedList.Item>
+      <OrderedList.Item>During the 1600s, tulips were worth more than gold</OrderedList.Item>
+    </OrderedList>
```

Comment:
There are two examples of ordered lists; they are identical. Should only be one?

Reply:
I think the second ordered list has increased sizing than the first ordered list. Each of the lists is sized as `small`, `medium`, and `large`, respectively.

Reply:
ah I see it now. This was very much not clear to me. Can we add a label to the examples that says something like, "Small ordered list" "Medium ordered list" and "Large unordered list"

Reply:
@allisonacs I can do that. @rossmoody would you be able to sync this change up with the Google Doc example images?

Reply:
It could be that there's too much going on in this example. Perhaps it would be clearer to break out one `ordered` vs `non-ordered` and another t-shirt size? Also, should we apply the `type="disc"` in order to indicate Garden's <ul> preference?

Reply:
I can add `type="disc"` in this PR 👍 . 

Regarding the example, maybe that can be addressed in a separate PR? As of now, this PR matches Google Doc spec. But I'd be happy to make the adjustments in this PR when the Google Doc is updated.